### DISCUSSION TITLE
BetterTTV for ReStreamio (OBS CSS Script Enabler)

### DISCUSSION THREAD

**Discussion by ShatterinSpider** [night/betterttv#5934] [Ideas]
I been wanting ReStream to intergrate BetterTTV since I started using it but it honestly doesn't seem like they will. Honestly I hope this would be a simple fix of using some sort of css script that would enable betterttv emotes to work with restream somehow and find some css script to do so but I'm no code expert.

**Comment by night** [comment]
CSS unfortunately does not load into the page the same way that JS does. You will need to continue asking the creator of that tool to support chat scripting like BetterTTV's.
